check.go

```diff
@@ -58,8 +86,13 @@ func checkMain(_ *cobra.Command, args []string) error {
 			return err
 		}
 
-		if licenseType == licenses.Forbidden {
-			fmt.Fprintf(os.Stderr, "Forbidden license type %s for library %v\n", licenseName, lib)
+		if hasLicenseNames && !isAllowedLicenseName(licenseName, allowedLicenseNames) {
+			fmt.Fprintf(os.Stderr, "Not allowed license %s found for library %v\n", licenseName, lib)
+			found = true
+		}
+
+		if hasLicenseType && isDisallowedLicenseType(licenseType, disallowedLicenseTypes) {
```

Comment:
See previous unmerged PR for a reference of how to handle unknown types: https://github.com/google/go-licenses/pull/44